traffic_control/management/commands/import_traffic_signs_vaisala.py

```diff
@@ -0,0 +1,63 @@
+import csv
+import os
+
+from django.conf import settings
+from django.contrib.gis.geos import Point
+from django.core.management.base import BaseCommand, CommandError
+
+from traffic_control.models import TrafficSignReal
+from traffic_control.models.traffic_sign import LocationSpecifier
+from users.utils import get_system_user
+
+SOURCE_NAME = "Vaisala 2019-2020"
+OWNER = "Helsingin kaupunki"
+SOURCE_SRID = 4326
+
+
+class Command(BaseCommand):
+    help = "Import vaisala traffic signs from a csv file"
+    step = 1000
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "filename", help="Path to the vaisala traffic sign csv file"
+        )
+
+    def handle(self, *args, **options):
+        filename = options["filename"]
+        if not os.path.exists(filename):
+            raise CommandError(f"File {filename} does not exist")
+
+        self.stdout.write("Importing vaisala traffic signs...")
+        count = 0
+        user = get_system_user()
+        with open(filename, encoding="utf-8-sig") as f:
+            csv_reader = csv.DictReader(f, delimiter=",")
+            for row in csv_reader:
+                location = Point(
+                    float(row["longitude"]), float(row["latitude"]), 0, srid=SOURCE_SRID
```

Comment:
Doesn't the `Point` need to be transformed from the source SRID to the SRID the platform uses?

Reply:
Yeah, it's done in next line

Reply:
TIL that apparently I'm blind. 🤦‍♂️